compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -362,6 +362,16 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
         }
     }
 
+    fn visit_generic_args(&mut self, args: &'a ast::GenericArgs) {
+        if let ast::GenericArgs::Parenthesized(generic_args) = args
```

Comment:
pls leave a comment explaining why this is being checked here

Reply:
Good point!